dae/dae/genomic_resources/repository.py

```diff
@@ -547,6 +547,8 @@ def check_update_manifest(
         manifest = Manifest()
         entries_to_update = set()
         for entry in self.collect_resource_entries(resource):
+            if entry.name.endswith("html"):  # Ignore generated info files

```

Comment:
I don't understand that if

Reply:
It was decided to ignore generated info pages in the manifest as their generation is fast and they are lightweight. If info pages are not ignored, then the manifest would also become immediately outdated the moment we run the info command.